src/TLuaInterpreter.cpp

```diff
@@ -12494,7 +12494,14 @@ void TLuaInterpreter::logError( std::string & e, const QString & name, const QSt
         mpHost->mpEditorDialog->mpErrorConsole->printDebug(green, black, s2 );
         mpHost->mpEditorDialog->mpErrorConsole->printDebug(red, black, s3 );
     }
-
+    
+    if( mpHost->mpConsole && mpHost->mEchoLuaErrors )
+    {
+    	  blue = QColor(80,80,255);
```

Comment:
Yeah. I was changing the color slightly just because I find dark blue extremely hard to read against a black background. I'm actually thinking about adding a preference for changing the default link color at some point just to make my life a little easier. Could easily remove that if you would prefer it the standard blue color, but it looks like your suggestion below will make this irrelevant anyway.